gemstore/index.js

```diff
@@ -1,9 +1,13 @@
 (function() {
-  var app = angular.module('gemStore', []);
+  var app = angular.module('gemStore', ['store-directives']);
 
-  app.controller('StoreController', function(){
-    this.products = gems;
-  });
+  app.controller('StoreController', ['$http', function($http){
+    var store =  this;
+    store.products = [];
+    $http.get('/store-products.json').success(function(data) {
+      store.products = data;
```

Comment:
@andrefyg the asset was added before on [d82dbb6](https://github.com/Manfred-Diaz/angular-training/commit/d82dbb61aa5affcfdeadc7b29f2f5640da3b8113)